lib/sles4sap/sap_deployment_automation_framework/configure_tfvars.pm

```diff
@@ -200,6 +204,39 @@ sub set_hana_db_parameters {
     set_var('SDAF_HANA_HA_SETUP', grep(/ha/, @{$args{components}}) ? 'true' : 'false');
 }
 
+=head2 set_fencing_parameters
+
+    set_fencing_parameters();
+
+Sets tfvars HA fencing related parameters according to scenario defined OpenQA settings.
+
+=cut
+
+sub set_fencing_parameters {
+    # Fencing mechanism AFA (Azure fencing agent - MSI), ASD (Azure shared disk - SBD), ISCSI (iSCSI based SBD fencing)
+    # Default value: 'msi' - AFA - Azure fencing agent (MSI)
```

Comment:
say there's a default but then `SDAF_FENCING_MECHANISM` is read with `get_required_var` and not with `get_var` with a default.

Reply:
Thanks, it's fixed now.